src/restconfig/src/main/java/org/geoserver/rest/catalog/WorkspaceController.java

```diff
@@ -112,8 +112,7 @@ public RestWrapper workspacesGet() {
 
         if (catalog.getWorkspaceByName(workspace.getName()) != null) {
             throw new RestException(
-                    "Workspace '" + workspace.getName() + "' already exists",
-                    HttpStatus.UNAUTHORIZED);
+                    "Workspace '" + workspace.getName() + "' already exists", HttpStatus.CONFLICT);
```

Comment:
This is a good thing to do, uses the HTTP error codes better.
However, it's no longer a documentation change, but a behavioral one, the second part of the PR checklist needs to be satisfied as well (e.g., Jira ticket, test, and so on)

Reply:
😀Thanks .

Reply:
> This is a good thing to do, uses the HTTP error codes better.
> However, it's no longer a documentation change, but a behavioral one, the second part of the PR checklist needs to be satisfied as well (e.g., Jira ticket, test, and so on)

I issue an issue [here](https://osgeo-org.atlassian.net/browse/GEOS-10007) and will continue working on this.